src/main/java/org/codeforamerica/shiba/pages/PageController.java

```diff
@@ -599,9 +599,9 @@ RedirectView submitFeedback(Feedback feedback,
       Locale locale) throws IOException, InterruptedException {
     LocaleSpecificMessageSource lms = new LocaleSpecificMessageSource(locale, messageSource);
     try {
-      applicationRepository.updateStatus(applicationData.getId(), UPLOADED_DOC,
-          routingDecisionService.getRoutingDestinations(applicationData, UPLOADED_DOC),
-          IN_PROGRESS);
+      routingDecisionService.getRoutingDestinations(applicationData, UPLOADED_DOC).forEach(
+          routingDestination -> documentStatusRepository.createOrUpdate(applicationData.getId(),
+              UPLOADED_DOC, routingDestination.getName(), IN_PROGRESS));
```

Comment:
Could we move some of this logic into DocumentStatusRepository? Probably a new method like `createOrUpdateAllForDocumentType()` but that only updates a single document type but does it for all routing destinations.